scripts/transfer_files.sh

```diff
@@ -3,54 +3,56 @@
 usage() {
   echo -n "
 #############################################################
```

Comment:
Why are these lines prefixed as if they were comments? It doesn't seem necessary.

Reply:
That's a good question. I've just copied and pasted over from, e.g. `take_bias.sh`, `take_pic.sh` but it does seem to be a little odd. I thought I originally copied that from one of your bash scripts but I can't see what that would have been.  In the `take_pic.sh` and `take_bias.sh` it reads like a docstring with examples below the comments.  Here the final `NOTE:` line breaks that pattern.